classes/specs/person_spec.rb

```diff
@@ -0,0 +1,44 @@
+# person_spec.rb
+
+require_relative '../person' # Adjust the path as needed
+require 'rspec'
+
+describe Person do
+  let(:person) { Person.new(name: 'John', age: 25, parent_permission: true) }
+
+  it 'adds a new rental to the rental list of a person' do
+    rental = double('Rental')
+    person.add_rental(rental)
+    expect(person.rentals).to include(rental)
+  end
+
+  it 'returns true if the person is of age' do
+    expect(person.can_use_services?).to be true
+  end
+
```

Comment:
- Hey, please, kindly add a test also for the `correct_name` method as this method is meant to be tested to meet the project requirement.

Reply:
Hi @zieeco , thank you so much for the feedback. we really appreciate.
Please the test for the correct_name was present already, I think you didn't notice it

https://github.com/ethred/OOP_schoo_library/blob/0a655d846a58ef90ffd508038a1f48f17a5f1f6a/classes/specs/person_spec.rb#L27-L29

thank you

Reply:
Test is added 👍🏿